### PR TITLE
[ecosystem] Use recaptcha v3 if possible & re-enable turbolinks.

### PR DESCRIPTION
**Pull request by zacharydenton**
## Motivation

Recaptcha v3 is invisible in most cases. It also works with turbolinks. Use it and fall back to recaptcha v2 if v3 fails.

captcha_v3

v2 fallback:

captcha_v2_fallback

### Have you read the [Contributing Guidelines on pull requests](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md#pull-requests)?

(You must have submitted a [signed CLA](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md#contributor-license-agreement) that includes your GitHub handle prior to us accepting and landing your work. Write your answer here.)

## Test Plan

Manual testing

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/aptos-labs/aptos-core/tree/main/developer-docs-site, and link to your PR here.)

### CODE REVIEW

File: ecosystem/platform/server/app/controllers/it1_profiles_controller.rb [hunk 1]
```diff
@@ -27,7 +27,7 @@ def create
     params[:user] = current_user
     @it1_profile = It1Profile.new(params)
 
-    respond_with(@it1_profile) and return unless verify_recaptcha(model: @it1_profile)
```
Comment (CapCap) [inline ecosystem/platform/server/app/controllers/it1_profiles_controller.rb:30]: Does this not also take an AR model?

**Review by CapCap — APPROVED** [review]
LGTM! Just a few lint things (unneeded paren)

**Comment by davidiw** [comment]
wow, I'm so envious, I had to fill out the captcha so many times... and many times it didn't actually even contain the image I was supposed to look for 😂

**Pull request merged**